source/_cookbook/track_battery_level.markdown

```diff
@@ -29,7 +29,30 @@ sensor:
 ```
 {% endraw %}
 
+### Android Devices
+
```

Comment:
On your Android device, once the official [Home Assistant companion app](https://companion.home-assistant.io/)  is  installed and connected to your Home Assistance instance, you will be able to display the battery level in the Frontend by adding a [template sensor](https://www.home-assistant.io/integrations/template) to your configuration yaml file.  You can also use the battery [sensor device class](https://www.home-assistant.io/integrations/sensor/#device-class) to dynamically change the icon with the battery level.

Reply:
I have added your text.